torch/multiprocessing/multiprocessing.md

```diff
@@ -0,0 +1,89 @@
+# CUDA IPC Refcounting implementation explained
```

Comment:
This doc feels kind of copy-pastey with the previous one. Would it not be possible to just write a single doc that addresses both situations?

Reply:
For example, you have the same precocious misspelling here

Reply:
Err, actually, this is an exact copy paste, isn't it

Reply:
removing this file.